src/meta/src/barrier/schedule.rs

```diff
@@ -315,6 +317,33 @@ impl BarrierScheduler {
         .map(|i| i[1])
     }
 
+    pub async fn run_command_until_collected(
```

Comment:
It is not clear the difference with `run_command` below, which `Run a command and return when it's completely finished.`  May you add some comments to elaborate the meaning of `until_collected`?

Reply:
Sorry, this PR is incomplete, and that code may just be deleted.